dev.env

```diff
@@ -10,4 +10,5 @@ DB_USER=
 DB_PASSWORD=
 DB_HOST=db
 DB_PORT=3306
+GITHUB_TOKEN=
```

Comment:
In here can you write Token's permission scope, please? It will be useful for us to generate it.